src/docs/patterns/appbar.mdx

```diff
@@ -140,7 +155,7 @@ For more advanced searching, you may wish to include an auto-complete feature. Y
 
 ## Mobile
 
-<ImageGrid images={[Specs1Image, Specs2Image, Specs3Image]} regularWidth />
+<ImageGrid images={[Specs1Image, Specs2Image, Specs3Image, Specs5Image, Specs6Image]} regularWidth />
```

Comment:
The 60% height / center labels on this guy are a little confusing. It might make more sense if the 60% line only extends down to the midline of the three liner instead of to the bottom of the app bar.